src/components/ReplyComment/index.js

```diff
@@ -129,7 +135,7 @@ const ReplyCommentId = ({
                           level={parseInt(level, 10) + 1}
                           updateVote={updateVote}
                           onLongPress={() => {
-                            showAlertDelete(itemReply, false, () => getThisComment(true));
+                            showAlertDelete(itemReply, false, updateComment);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `showAlertDelete` function now receives `updateComment` as a third argument. Ensure that `updateComment` behaves as expected.
```diff
-                            showAlertDelete(itemReply, false, () => getThisComment(true));
+                            showAlertDelete(itemReply, false, updateComment);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->